src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -131,7 +132,8 @@ def collect_data(geop_result, geojson):
     z['n42b'] = round(z['StreamLength'] / 1000, 1)  # Kilometers
 
     # Data from Point Source Discharge dataset
-    n_load, p_load, discharge = point_source_discharge(geom, area)
+    n_load, p_load, discharge = point_source_discharge(geom, area,
+                                                       in_drb=geom.within(DRB))
```

Comment:
Consider following previously established convention of naming this parameter `drb`. See [stream_length](https://github.com/WikiWatershed/model-my-watershed/blob/c2eacd3383753a51badfed30190dab33e3e4e03b/src/mmw/apps/modeling/mapshed/calcs.py#L301) and [streams](https://github.com/WikiWatershed/model-my-watershed/blob/c2eacd3383753a51badfed30190dab33e3e4e03b/src/mmw/apps/modeling/mapshed/calcs.py#L324).